xml/schema/Contribute/ContributionSoft.xml

```diff
@@ -129,6 +129,9 @@
     <type>varchar</type>
     <title>Soft Contribution PCP Note</title>
     <length>255</length>
+    <html>
+      <type>Text</type>
```

Comment:
@eileenmcnaughton should the type not be TextArea?

Reply:
hmm - maybe - why was it text on the other note field - yes will change

Reply:
done

Reply:
I think that was a text field where as this is a rich text field perhaps?